src/helper_gitlab_modify.sh

```diff
@@ -385,10 +384,11 @@ delete_gitlab_repository_if_it_exists() {
 
 
 #######################################
-# Checks if repository exists in the GitLab server and creates a new empty repository 
+# Checks if repository exists in the GitLab server and creates a new empty repository
```

Comment:
The accuracy of this description can be improved. The function does not check whether the repository exists. Instead, it deletes the repository if it exists, and throws an error otherwise.